gdraw/gfilechooser.c

```diff
@@ -460,6 +486,29 @@ static void GFileChooserScanDir(GFileChooser *gfc,unichar_t *dir) {
 	ti = malloc((cnt+1)*sizeof(GTextInfo *));
 	tot = cnt-1;
     }
+
+#ifdef _WIN32
+    ti = realloc(ti, (dcnt+cnt+1)*sizeof(GTextInfo *));
+
+    unichar_t buffer[1025];
+    // Find "root" (e.g. C) and...
+    char win32root = toupper(u_GFileGetAbsoluteName(dir,buffer,sizeof(buffer)/sizeof(unichar_t))[0]);
```

Comment:
I don't think it's possible for `dir` to not be absolute (i.e. there shouldn't be any need to call `u_GFileGetAbsoluteName`)